ProcessLib/HydroMechanics/HydroMechanicsFEM-impl.h

```diff
@@ -53,10 +54,12 @@ HydroMechanicsLocalAssembler<ShapeFunctionDisplacement, ShapeFunctionPressure,
                           IntegrationMethod, DisplacementDim>(
             e, is_axially_symmetric, _integration_method);
 
+    auto& solid_material = MaterialLib::Solids::selectSolidConstitutiveRelation(
```

Comment:
const?

Reply:
This is actually possible. Did mark the integrateStress and create*State functions as const, then this one and in the IPData too...